cleanlab/rank.py

```diff
@@ -652,3 +653,106 @@ def get_outlier_scores(
         return (outlier_scores, knn)
     else:
         return outlier_scores
+
+
+def get_ood_scores(
+    pred_probs: np.array,
+    confident_thresholds: Optional[np.ndarray] = None,
+    labels: Optional[np.ndarray] = None,
+    adjusted_pred_probs: bool = False,
+    scoring_method: str = "entropy",
```

Comment:
shorten argname to `method` to better match other functions in this file

Reply:
Prioritize same file format > other file formats?

Reply:
yep I think that's a reasonable heuristic :)